Meadowlark-js/backends/meadowlark-opensearch-backend/src/repository/UpdateOpensearch.ts

```diff
@@ -38,8 +65,7 @@ export async function afterDeleteDocumentById(request: DeleteRequest, result: De
     );
     await client.delete({ ...opensearchRequest, refresh: true });
   } catch (err) {
-    Logger.error(`${moduleName}.afterDeleteDocumentById`, request.traceId, err);
-    throw err;
+    throw safelyLogOpenSearchErrors(err);
```

Comment:
This is difficult to test. I will see if I can come up with a manual test.

Reply:
I haven't been able to figure anything out yet. I think this code is written in a very safe way - but it needs careful visual scrutiny.

Reply:
I'm completely rolling back changes to this file. Other backend modules also have this double logging problem, and I'll create a new ticket to address that.